DropboxSync.BLL/DropboxSyncContext.cs

```diff
@@ -0,0 +1,44 @@
+﻿using DropboxSync.BLL.Configurations;
+using DropboxSync.BLL.Entities;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DropboxSync.BLL
+{
+    public class DropboxSyncContext : DbContext
+    {
+        public DbSet<DossierEntity> Dossiers { get; set; }
+        public DbSet<ExpenseEntity> Expenses { get; set; }
+        public DbSet<InvoiceEntity> Invoices { get; set; }
+        public DbSet<UploadEntity> Uploads { get; set; }
+
+#pragma warning disable CS8618 // Un champ non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le champ comme nullable.
+        public DropboxSyncContext()
+#pragma warning restore CS8618 // Un champ non-nullable doit contenir une valeur non-null lors de la fermeture du constructeur. Envisagez de déclarer le champ comme nullable.
+        {
+
+        }
+
+        protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
+        {
+            optionsBuilder.UseSqlite("Filename=DropboxSyncDatabase.db", options =>
```

Comment:
if possible, it would be nice that these things can be set as env variables
it would be nice if we can switch from sqlite to e.g postgres in the future, without recompiling the code
same for the dbname, should be an env variable with a default value

Reply:
The migration from SQLite to PostgreSQL is doable but it requires new Nugget Package and a different configuration like mentioned [here](https://www.npgsql.org/efcore/). It is 90% the same.